app.py

```diff
@@ -149,6 +153,35 @@ def get_book_info():
     )
 
 
+@app.route("/get_favorites", method="POST")
```

Comment:
Favorites route already exists that grabs all of the current user's favorite ISBNs and passes specific info about each book to the favorites.html page to render. This can be deleted.